pkg/helpers/plugins.go

```diff
@@ -0,0 +1,37 @@
+package helpers
+
+import (
+	"fmt"
+
+	"github.com/safedep/code/core"
+	"github.com/safedep/code/fs"
+	"github.com/safedep/code/lang"
+	"github.com/safedep/code/parser"
+)
+
+func SetupBasicPluginContext(filePaths []string, languageCode core.LanguageCode) (core.TreeWalker, core.ImportAwareFileSystem, error) {
```

Comment:
If this is used only in tests, lets move this to `pkg/test/helpers`. Otherwise this becomes an exported API that we have to support. This seems like very use-case specific helper where we are explicitly creating a tree walker.

Reply:
ohh, alright
will keep this in mind for any `test` utils if needed in future